fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_life.dm

```diff
@@ -374,12 +373,11 @@
 			/// Otherwise, check if it's Sol, to enter Torpor.
 			if(clan.bloodsucker_sunlight.amDay)
 				Check_Begin_Torpor(TRUE)
-		/// You are in Torpor, and in a Coffin. Check if it's not Daytime & you have less than 10 Brute/Burn combined to end Torpor.
-		else if(!clan.bloodsucker_sunlight.amDay && total_damage <= 10)
+		/// You are in Torpor, and in a Coffin. Check if you are supposed to end torpor
+		else
 			Check_End_Torpor()
-	/// You're not in a Coffin, but are in Torpor. Check if it's not Daytime, & you have less than 10 Brute (NOT Burn) to end Torpor.
-	else if(!clan.bloodsucker_sunlight.amDay && total_brute <= 10 && HAS_TRAIT(owner.current, TRAIT_NODEATH))
-		Check_End_Torpor()
+	/// You're not in a Coffin, but are in Torpor. Check if you are supposed to end torpor
+	Check_End_Torpor()
```

Comment:
are you just straight up calling it twice every single lifetick

Reply:
I realized later that the previous one isn't necessary, and I'll just have this one